psalm.xml

```diff
@@ -2,6 +2,7 @@
 <psalm
   errorLevel="1"
   strictBinaryOperands="true"
+  findUnusedBaselineEntry="true"
```

Comment:
Looks like a useful addition, though on just reading the [one-line description](https://psalm.dev/docs/running_psalm/configuration/#findunusedbaselineentry), I don't fully understand what it actually does.

Reply:
This switch controls whether Psalm will complain about baseline entries listing a problem that does not exist (anymore).

Reply:
Ah, so some of the warnings that were removed in this PR should perhaps have been removed previously.

Reply:
Yes, possibly (though I tried to remember to update the baseline file when updating Psalm).